README.md

```diff
@@ -15,133 +16,93 @@ Some feature highlights:
-
-For instructions on how to build Drawpile on Windows and OSX, see the [Building from sources] page.
-
-[QtColorPicker]: https://gitlab.com/mattia.basaglia/Qt-Color-Widgets
-[QtKeyChain]: https://github.com/frankosterfeld/qtkeychain
-
-### Building on Linux
-
-This is how to build on a plain Ubuntu 22.04. Other Linuxes are similar, the dependencies just have different names. There's two options: Qt6 or Qt5. Pick one of them, don't install both. If you're using an older version of your distribution, Qt6 might not yet be available for you.
-
-Here's how to build with Qt6:
-
-```sh
-# Install dependencies (Qt6)
-apt install -y build-essential cmake extra-cmake-modules git libgles2-mesa-dev libjpeg-dev libmicrohttpd-dev libpng-dev libsodium-dev libqt6svg6-dev libvulkan-dev libxkb-common-dev libz-dev libzip-dev ninja-build qtkeychain-qt6-dev qt6-multimedia-dev qt6-base-dev qt6-l10n-tools qt6-tools-dev qt6-tools-dev-tools zipcmp zipmerge ziptool
-
-# Clone the repositories and check out the correct branches.
-git clone --branch dev-2.2 https://github.com/drawpile/Drawdance.git
-git clone --branch dancepile-2.2 https://github.com/askmeaboutlo0m/Drawpile.git
-
-# Build Drawdance
-cd Drawdance
-scripts/cmake-presets drawpile_qt6_release
-cmake --build builddpqt6release
-
-# Build Drawpile
-cd ../Drawpile
-scripts/cmake-presets drawpile_qt6_release
-cmake --build builddpqt6release
-
-# Install Drawpile - this is required to be done at least once! Do NOT just run
-# Drawpile from the build directory, you'll be missing brushes, themes and other
-# stuff. If you did it anyway, installing won't fix it, you'll have to delete
-# ~/.local/share/drawpile/drawpile and maybe ~/.config/drawpile/drawpile.ini.
-sudo cmake --install builddpqt6release
-
-# Run Drawpile
-drawpile
-```
-
-And here with Qt5:
-
-```sh
-# Install dependencies (Qt5)
-apt install -y build-essential cmake extra-cmake-modules git libjpeg-dev libkf5archive-dev libmicrohttpd-dev libpng-dev libsodium-dev libqt5svg5-dev libz-dev ninja-build qt5keychain-dev qt5multimedia-dev qtbase5-dev qttools5-dev
-
-# Clone the repositories and check out the correct branches.
-git clone --branch dev-2.2 https://github.com/drawpile/Drawdance.git
-git clone --branch dancepile-2.2 https://github.com/askmeaboutlo0m/Drawpile.git
-
-# Build Drawdance
-cd Drawdance
-scripts/cmake-presets drawpile_release
-cmake --build builddprelease
-
-# Build Drawpile
-cd ../Drawpile
-scripts/cmake-presets drawpile_release
-cmake --build builddprelease
-
-# Install Drawpile - this is required to be done at least once! Do NOT just run
-# Drawpile from the build directory, you'll be missing brushes, themes and other
-# stuff. If you did it anyway, installing won't fix it, you'll have to delete
-# ~/.local/share/drawpile/drawpile and maybe ~/.config/drawpile/drawpile.ini.
```

Comment:
For Android and macOS the assets are copied at build time by CMake so it is fine. For Linux and Windows, probably the thing to do is to just generate a qt.conf into the drawpile `RUNTIME_OUTPUT_DIRECTORY` which contains `[Paths] Data = path/to/assets` so when someone runs the application from there it will just work. I will make this change.

Reply:
Turns out that was wrong and didn’t work, but there is a path to making this work later. Once changes I’ve already made to support theme switching for `QIcon` are landed, then the paths that are used for lookups of everything can be changed to also search for the required files in an arbitrary assets directory, and then it will all work OK.